packages/web-components/@groupby/elements-autocomplete/test/unit/autocomplete.test.ts

```diff
@@ -432,4 +433,26 @@ describe('Autcomplete Component', () => {
       expect(autocomplete.selectedIndex).to.equal(newSelectedIndex);
     });
   });
+
+  describe('sendUpdateSearchEvent()', () => {
+    it('should send a search request with a search term', () => {
+      const term = 'autocomplete-term';
+      const clickEvent = {
+        target: {
+          innerText: term,
+        },
+      };
+      const group = autocomplete.group = 'group1';
+      const payload = {
+        term,
+        group,
+        search: true,
+      };
+      const dispatchElementsEvent = stub(autocomplete, 'dispatchElementsEvent');
+
+      autocomplete.sendUpdateSearchEvent(clickEvent);
+
+      expect(dispatchElementsEvent).to.be.calledWith(UPDATE_SEARCH_TERM, payload);
+    });
+  });
```

Comment:
Is this dispatching an UPDATE_SEARCH_TERM event rather than sending a search request as the test description states?

Reply:
Updated the test description to better reflect that.